### DISCUSSION TITLE
adding more than 1 assets at a time

### DISCUSSION THREAD

**Discussion by jcv99** [requarks/wiki#3979] [Help / Questions]
I would love to know if there's a method to put more than 1 asset, I have to move a 1000 pdf and if i could do at least 10 by 10 I would be 10 times faster

**Comment by skledarg** [comment]
Yes, you can change the `Max Files per Upload` in the web administration interface (under Security).  If you're using nginx or another reverse proxy in front of Wikijs, you might also need to change the configuration there.

**Reply by jcv99** [reply]
My question was the other way. How can i put more assets in the page. Not upload them

**Reply by skledarg** [reply]
Currently you can only select one asset in the asset browser - there is a feature request at https://js.wiki/feedback/p/insert-multiple-assets-at-once
